src/components/ObjectInfo.js

```diff
@@ -0,0 +1,35 @@
+function ObjectInfo({arrangement, onItemPress, side, style}) {
+
+    function onPress({idbtn, gender:gender }) {
+        onItemPress({side, id:idbtn, gender:gender});
+    }
+
+    return (
+        <li className={'UserInfo'+side} style={style}>
```

Comment:
```suggestion
        <li className={`UserInfo${side}`} style={style}>
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals